pkg/sidecar-controller/snapshot_controller_base.go

```diff
@@ -124,12 +123,7 @@ func NewCSISnapshotSideCarController(
 				// So we are skipping the re-queue here to avoid CreateSnapshot being called without exponential backoff.
 				newSnapContent := newObj.(*crdv1.VolumeSnapshotContent)
 				if newSnapContent.Status != nil && newSnapContent.Status.Error != nil {
-					oldSnapContent := oldObj.(*crdv1.VolumeSnapshotContent)
```

Comment:
Can you update the comments accordingly.

Reply:
Done @xing-yang let me know if the comments are ok